tests/test_lowlevel.py

```diff
@@ -2562,8 +2562,9 @@ def test_keep_mutations_block_size_mutations(self):
         tables = tskit.TableCollection(1)
         tables.nodes.add_row(0)
         tables.sites.add_row(0, "a")
-        for _ in range(8192):
```

Comment:
I don't really know what this block size thing is doing, but note it's changed from 8192 mutations to three. (If 8192 is important, you could do
```
         for _ in range(4096):
             tables.mutations.add_row(0, node=0, time=0, derived_state="b")
             tables.mutations.add_row(0, node=0, time=0, derived_state="c")
```